tools/pipelines/test-dds-stress.yml

```diff
@@ -59,9 +59,52 @@ parameters:
     testFileTarName: matrix
     testCommand: test:stress
 
+
 stages:
   - template: templates/include-conditionally-run-stress-tests.yml
     parameters:
       artifactBuildId: $(resources.pipeline.client.runID)
       packages: ${{ parameters.packages }}
       testWorkspace: ${{ variables.testWorkspace }}
+      stressMode: ${{ parameters.stressMode }}
+
+  # Capture telemetry about pipeline stages
+  - stage: upload_run_telemetry
```

Comment:
seems like a bad merge with Alejandro's recent change to refactor this. I would have only expected a change to `stressMode:` in this PR